script.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+//Create 2 objects: animal and cat, add move property to animal object, cat object must inherit move property from object animal
+let animal = [];
+let cat = []
+animal.canMove = true ;
+console.log(animal);
+Object.assign(cat , animal );
+console.log(cat);
```

Comment:
assign method copies all enumerable own properties from one or more source objects to a target object. It returns the modified target object. it's not inheritance